src/php/lib/Grpc/AbstractCall.php

```diff
@@ -33,7 +33,7 @@
  */
 namespace Grpc;
 
-require_once realpath(dirname(__FILE__) . '/../autoload.php');
+require_once realpath(dirname(__FILE__) . '/../../vendor/autoload.php');
```

Comment:
I guess it can be removed, but not because of anything special composer does. Composer just generates the `autoload.php` file that I refer to there. If we assume that these library files will only be loaded using that autoload file, then the `require_once` line is redundant. That probably is a safe assumption, so I'll remove it from files in `lib/Grpc`.